### ISSUE TITLE
ciHelper::updateCommitMappings Failed

### ISSUE THREAD

**Issue by webstech** [gitgitgadget/gitgitgadget#218]
In [pipeline job](https://dev.azure.com/gitgitgadget/git/_build/results?buildId=35906&view=logs&j=fd490c07-0b22-5182-fac9-6d67fe1e939b&t=59d5b2ff-d842-581a-8fe3-fd774adf34a7&l=38), it failed for [git pr 546](https://github.com/git/git/pull/546).  The meta data head commit was not found.  Need to determine how this should be handled.  @dscho Is this a problem with the new dugite (did not throw an error before)?
```
node build\script\misc-helper.js get-pr-meta -g ..\git git 546
{
    "baseCommit": "0ad714499976290d9a0229230cbe4efae930b8dc",
    "baseLabel": "git:maint",
    "coverLetterMessageId": "pull.546.git.git.1581479550353.gitgitgadget@gmail.com",
    "headCommit": "564506e0556b93dda56cc0f77734bb46feba9c9e",
    "headLabel": "gstrauss:lighttpd-git-http-backend",
    "iteration": 1,
    "latestTag": "pr-git-546/gstrauss/lighttpd-git-http-backend-v1",
    "pullRequestURL": "https://github.com/git/git/pull/546"
}
```

**Comment by dscho** [comment]
Hmm... those commits should be there. Maybe we didn't fetch in the git worktree?

**Comment by webstech** [comment]
Is there a way to look at the repo in the pipeline job?  If (a big if) I understand this, the commit should have been fetched when the submit was done on Feb 11.

**Comment by dscho** [comment]
In Attempt no. 3, it finally worked.

**Comment by webstech** [comment]
Thanks for the extra diagnosis on this.  Git 2.23 is not all that old.  Does the seg-fault occur running the  git command by itself or only with dugite (did not think the range-diff would be very big)?  Not really expecting you to test this (could try myself).  Bottom line is, do we need to do more diagnosis, open a dugite issue, open a git issue or just ignore and use the more current version of git?  Not making light of it - more diagnosis is what I am used to doing - just wanting to know what is the usual process.

**Comment by dscho** [comment]
It was a bug in `range-diff` IIRC, and yes, I tested this from the Bash, side-stepping dugite.

**Comment by dscho** [comment]
I'll ping the dugite maintainers about the issue.

**Comment by dscho** [comment]
Dugite v1.91.0 was just released, it should come with v2.26.0...
